internal/cloud/realm/client.go

```diff
@@ -35,12 +58,36 @@ func (c *client) do(method, path string, options api.RequestOptions) (*http.Resp
 		return nil, err
 	}
 
-	req.Header = options.Header
-	if req.Header == nil {
-		req.Header = http.Header{}
-	}
+	req.Header = http.Header{}
 	req.Header.Set(requestOriginHeader, cliHeaderValue)
 
+	if options.ContentType != "" {
+		req.Header.Set(api.HeaderContentType, options.ContentType)
+	}
+
+	if auth, err := c.getAuth(options); err != nil {
+		return nil, err
+	} else if auth != "" {
+		req.Header.Set(api.HeaderAuthorization, "Bearer "+c.session.AccessToken)
```

Comment:
you need to use `auth` instead of `c.session.AccessToken` here

Reply:
😬 Thanks, great catch